CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog 
 
+## 1.52.0
+
+- database library changed from rusqlite to sqlx #2089 #2331 #2336 #2340
+
+- add alias support: UIs should check for `dc_msg_get_override_sender_name()`
+  also in single-chats now and display divergent names and avatars #2297
+
+- parse blockquote-tags for better quote detection #2313
+
+- ignore unknown classical emails from spam folder #2311
+
+- support "Mixed Up” encryption repairing draft #2321
```

Comment:
I would drop "draft" here, it's unclear that this is about RFC draft. "Mixed Up" encryption is repaired, that's the most important.

Reply:
k, i change that.